pkg/apis/stork/v1alpha1/types.go

```diff
@@ -10,6 +11,14 @@ type RuleActionType string
 const (
 	// RuleActionCommand is a command action
 	RuleActionCommand RuleActionType = "command"
+	// ClusterPairResourceName is name for "clusterpair" resource
```

Comment:
Will commit the CRD changes first when PR is approved so that sched-ops can be updated followed by govendor update here.